samples/net/mqtt_publisher/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Config options for mqtt_publisher sample application
+
+# Copyright (c) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Networking mqtt_publisher sample application"
+
+config APP_MAX_ITERATIONS
```

Comment:
Missed this one. We have actually called these sample app variables like this `NET_SAMPLE_...` in other samples so we could follow the same style.

Reply:
Fixed.